config/default/manager_auth_proxy_patch.yaml

```diff
@@ -15,7 +15,7 @@ spec:
           capabilities:
             drop:
             - "ALL"
-        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.15.0
+        image: "rbac_proxy"
```

Comment:
@danielpodwysocki 
Hi, I didn't make any test for this PR locally, but I think this would break _non-helm_ based installation.
This `image` param will be used as-is in both `make deploy` and `kubectl apply -k.` ways that described in _Basic install_ section and will cause `ErrImagePull` error: https://ansible.readthedocs.io/projects/awx-operator/en/latest/installation/basic-install.html
If you want to replace this `image` with `rbac_proxy`, further modifications should be made so that the default image is correctly specified for non-helm installations.

Reply:
Thank you, wasn't aware you were also handling non-helm installs there - will circle back to this soon-ish, make it work and update the PR.

Reply:
I think to make this work for normal operator installs (non-helm), you would need to add an entry to this kustomization file:
* https://github.com/ansible/awx-operator/blob/0d1fa239a56c55202167cb833a72f495928442fc/config/default/kustomization.yaml

that is similar to the existing one for the operator image, here:
* https://github.com/ansible/awx-operator/blob/devel/config/manager/kustomization.yaml#L12-L17

Reply:
I have updated the PR to not break Kustomize - apologies it took a bit, life is  sometimes very good at getting in the way of things :)

Would you mind having another look? I also made imagePullSecrets customizable (while keeping the default value).

Reading the code for the operator itself, the image_pull_secrets var defined in `roles/installer/defaults/main.yml` takes care of things already there, so that shouldn't need extra handling. 

And thank you once more for having a look!

PS. quick test I did:
```
# deploying to a local dev cluster running in kind
➜  awx-operator git:(1301-parametrize-image-locations) ✗ make deploy
namespace/awx unchanged
customresourcedefinition.apiextensions.k8s.io/awxbackups.awx.ansible.com unchanged
customresourcedefinition.apiextensions.k8s.io/awxmeshingresses.awx.ansible.com unchanged
customresourcedefinition.apiextensions.k8s.io/awxrestores.awx.ansible.com unchanged
customresourcedefinition.apiextensions.k8s.io/awxs.awx.ansible.com unchanged
serviceaccount/awx-operator-controller-manager unchanged
role.rbac.authorization.k8s.io/awx-operator-awx-manager-role configured
role.rbac.authorization.k8s.io/awx-operator-leader-election-role unchanged
clusterrole.rbac.authorization.k8s.io/awx-operator-metrics-reader unchanged
clusterrole.rbac.authorization.k8s.io/awx-operator-proxy-role unchanged
rolebinding.rbac.authorization.k8s.io/awx-operator-awx-manager-rolebinding unchanged
rolebinding.rbac.authorization.k8s.io/awx-operator-leader-election-rolebinding unchanged
clusterrolebinding.rbac.authorization.k8s.io/awx-operator-proxy-rolebinding unchanged
configmap/awx-operator-awx-manager-config unchanged
service/awx-operator-controller-manager-metrics-service unchanged
deployment.apps/awx-operator-controller-manager unchanged
➜  awx-operator git:(1301-parametrize-image-locations) ✗ kubectl describe deployments.apps -n awx awx-operator-controller-manager  | grep -i image
    Image:      gcr.io/kubebuilder/kube-rbac-proxy:v0.15.0
    Image:      quay.io/ansible/awx-operator:2.11.0-7-g3d080f0
```